tests/basic-test/basic-test.py

```diff
@@ -0,0 +1,93 @@
+from autogen_agentchat.agents import AssistantAgent
+from autogen_agentchat.teams import RoundRobinGroupChat
+from autogen_agentchat.conditions import TextMentionTermination
+from autogen_agentchat.ui import Console
+
+from autogen_ext.models.ollama import OllamaChatCompletionClient
+from autogen_ext.tools.mcp import SseMcpToolAdapter, SseServerParams, mcp_server_tools
+
+from pydantic import BaseModel
+
+import asyncio
+
+# Define output structure for the script
+class ScriptOutput(BaseModel):
+    topic: str
+    takeaway: str
+    captions: list[str]
+
+async def main():
+
+    # Initialize Ollama client (if needed)
+    ollama_client = OllamaChatCompletionClient(
+        # model="granite3.2:8b-instruct-q4_K_M",
+        model="llama3.1:latest",
+        api_key="placeholder",  # Placeholder API key for local model
+        response_format=ScriptOutput,
+        base_url="http://localhost:11434/v1",
+        model_info={
+            "function_calling": True,
+            "json_output": True,
+            "vision": False,
+            "family": "unknown"
+        }
+    )
+
+    # Create server params for the remote MCP service
+    server_params = SseServerParams(
+        url="http://localhost:8080/mcp/sse"
```

Comment:
how is wanaku started?

It seems like that you would like to test http provider (dog, meow facts), do you start only one provider?

Reply:
You have to start it manually (i.e.: within IntelliJ). The tool just automate the interaction ...